aftership-sdk/src/main/java/com/aftership/sdk/utils/JsonUtils.java

```diff
@@ -29,6 +29,7 @@ private static Gson createGson() {
     GsonBuilder builder = new GsonBuilder();
     builder.registerTypeAdapter(Date.class, new GsonDateDeSerializer());
     builder.setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES);
+    builder.serializeNulls();
```

Comment:
will this be used in PATCH requests? If so it would unexpectedlly add `null` values to the requests.

Reply:
yes.

Reply:
will remove it

Reply:
done